app/src/main/java/org/hackillinois/android/notifications/HackIllinoisNotificationManager.kt

```diff
@@ -0,0 +1,81 @@
+package org.hackillinois.android.notifications
+
+import android.app.*
+import android.content.Context
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.os.Build
+import org.hackillinois.android.R
+import org.hackillinois.android.database.entity.Event
+import java.util.*
+
+class HackIllinoisNotificationManager {
```

Comment:
It would be nice if we could call this something shorter... is `NotificationManager` already taken?

Reply:
Yeah it's already taken. I was thinking that maybe we should remove local notifications though? There's already functionality for remote push notifications and event favoriting in the API. Notifications are just a huge pain locally and would remove a lot of confusing code that does stuff on bootup.